src/components/Modals/DeleteExpenseModal.js

```diff
@@ -19,14 +17,16 @@ const DeleteExpenseModal = forwardRef((props, ref) => {
     },
   }));
 
-  const title = expense.title || '';
-  const date = expense.date ? getUKFormattedDate(expense.date) : '';
-  const amount = expense.amount ? getUKFormattedEuros(expense.amount) : '';
+  if (expense) {
```

Comment:
i am not sure, but i think the assignment/existence of these variables is not necessary. maybe checkout optional chaining (a.k.a. safe navigator): https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html